dev/src/graphics.cpp

```diff
@@ -945,13 +945,15 @@ nfr("render_mesh", "m", "R:mesh", "",
         return NilVal();
     });
 
-nfr("save_mesh", "m,name", "R:meshS", "B",
-    "saves the specified mesh to a file in the PLY format. useful if the mesh was generated"
-    " procedurally. returns false if the file could not be written",
-    [](StackPtr &, VM &vm, Value &i, Value &name) {
+nfr("save_mesh", "m,name,format", "R:meshSI?", "S?",
+    "saves the mesh to a file in the specified format (iqm by default). for other "
```

Comment:
Since previously this function saved PLY, making PLY 0 (default) and IQM 1 makes it compatible with existing code